### DISCUSSION TITLE
Allow hiding unusued issue labels in dropdown on Issues (search) page

### DISCUSSION THREAD

**Discussion by Ascor8522** [community/community#39188] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

Hello 👋 

When searching for issues in a repo, you can filter them by label (using the label dropdown).
Unfortuantely, lots of projects tend to create lots of labels, and most of the time, some of them are left unused.

__Consequences:__
This is creating clutter when searching for good issues to work on when searching by label.

__Practical Example:__
You pick a label that seems interesting to work on, but then find there are no open issues related to that label. 🙃 

__Suggestion:__
Please allow hiding all the labels that aren't associated with any open issue.

**Comment by david-engelmann** [comment]
Would this not have a negative effect on searching for closed issue with less frequently used tags?
